interfaces/tracing/v0/README.md

```diff
@@ -0,0 +1,67 @@
+# `tracing`
+
+## Usage
+
+This relation interface is meant to exchange the necessary information for a charm to integrate with a Tempo-compliant tracing backend.
```

Comment:
It's not just for the charm that integrates with a tempo-compliant tracing backend, it's also for that tempo-compliant tracing backend. I think the generic `This relation interface describes the expected behavior of any charm claiming to be able to provide or consume a tempo-compliant tracing server.`

Reply:
Agreed.